src/opengl/text.rs

```diff
@@ -299,7 +299,7 @@ impl TextRenderer {
     where
         S: Surface,
     {
-        let [x, y] = pos;
+        let (x, y) = (pos[0], pos[1]);
```

Comment:
Is there a reason for this more verbose form?

Reply:
I wanted to let it compile on stable, which doesn't have slice pattern syntax